package.json

```diff
@@ -218,7 +218,7 @@
             },
             {
                 "command": "appService.DisableRemoteDebug",
```

Comment:
I'd prefer if you made the command name consistent with 'Stop' as well

Reply:
What about the telemetry event?

Reply:
What about it?